packages/strapi-plugin-upload/admin/src/components/Flex/index.js

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+const Flex = styled.div`
+  display: flex;
```

Comment:
It is ok for the moment. However, this component should be improved with some props like justifyContent, alignItems, flexDirection etc

Reply:
Sure! Let's gradually improve it when needed.